cypress/pages/workflow.ts

```diff
@@ -24,6 +24,9 @@ export class WorkflowPage extends BasePage {
 		ndvParameterInput: (parameterName: string) =>
 			cy.getByTestId(`parameter-input-${parameterName}`),
 		ndvOutputPanel: () => cy.getByTestId('output-panel'),
+		ndvRunDataPaneHeader: () => cy.getByTestId('run-data-pane-header'),
+
+		successToast: () => cy.get('.el-notification__title'),
```

Comment:
How unique is this? Is there a more reliable way to check if the node is executed successfully?

Reply:
Not sure I understand. `successToast` is generic and gets narrowed down with `.contains('Node executed successfully')`. Can you explain?

Reply:
I am worried about selecting it using  the (`.el-notification__title`) class, looks a bit generic to me. If it ok on your side, go for it.